lhotse/dataset/cut_transforms/perturb_vol.py

```diff
@@ -0,0 +1,34 @@
+import random
+from typing import List, Sequence, Union
+
+from lhotse import CutSet
+
+
+class PerturbVol:
+    """
+    A transform on batch of cuts (``CutSet``) that perturbs the volume of the recordings
+    with a given probability :attr:`p`.
+
+    If the effect is applied, then one of the perturbation factors from the constructor's
+    :attr:`factors` parameter is sampled with uniform probability.
+    """
+
+    def __init__(
+            self,
+            factors: Union[float, List[float]],
```

Comment:
```suggestion
            factors: Union[float, Sequence[float]],
```
This allows tuples without type checker conflicts